qoqo/src/operations/pragma_operations.rs

```diff
@@ -1210,6 +1210,22 @@ pub struct PragmaConditional {
     circuit: Circuit,
 }
 
+#[wrap(Operate, OperatePragma)]
+/// A circuit controlled by a qubit.
+///
+/// The circuit is applied when the qubit is in state 1.
+/// Note that this is a unitary operation (for example a CNOT(0,1)
+/// is equvalent to a PragmaControlledCircuit(0, [PauliX(1)]) but it cannot be represented
+/// by a unitary operation in qoqo for arbitraty circuits.
+///
+/// Args:
+///     controlling_qubit (int): - The qubit controlling circuit application.
+///     circuit (Circuit): - The circuit executed if the condition is met.
+pub struct PragmaControlledCircuit {
```

Comment:
The name should be added to the mod.rs of operations: https://github.com/HQSquantumsimulations/qoqo/blob/main/qoqo/src/operations/mod.rs#L134  with a comment about it being in version 1.5

Reply:
Good catch thanks.